backend/open_api_spec.yml

```diff
@@ -0,0 +1,790 @@
+openapi: 3.0.0
+info:
+servers:
+paths:
+  /curation/auth/key:
+    post:
+      summary: Generate the curator's API key
+      security:
+        - cxguserCookie: []
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  key:
+                    type: string
+                    required: true
+                  id:
+                    type: string
+                    required: true
+        "401":
+          $ref: "#/components/responses/401"
+    get:
+      summary: Fetch the identifier of the curator's active API keys
+      security:
+        - cxguserCookie: []
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: string
+        "401":
+          $ref: "#/components/responses/401"
+    delete:
+      summary: Delete the curator's API key
```

Comment:
This is an internal portal API.

Reply:
This and all of the key and token APIs will be mainly portal specific, but they still need to be defined somewhere. We can move them to the Other API later.

Reply:
I think if we just flag it as "internal" in the description, then there's more awareness for reviewers until we can offer a demonstration notebook with step by step instructions?

Reply:
Ok, will add flag

Reply:
Maybe we could use tags for now:
- https://swagger.io/docs/specification/grouping-operations-with-tags/

Reply:
Tag could be in addition, but we should put internal in the Summary and or Description.